operator/Makefile

```diff
@@ -340,6 +340,10 @@ deploy-via-olm: ## Deploy operator image to the cluster using OLM.
 # target(s) that do it here, because CI does it in a separate job, in parallel with cluster deployment.
 	KUTTL=$(KUTTL) ./hack/olm-operator-install.sh $(TEST_NAMESPACE) $(IMAGE_TAG_BASE) $(VERSION)
 
+.PHONY: deploy-via-olm-midstream
+deploy-via-olm-midstream: ## Deploy operator image to the cluster using OLM and midstream index images.
```

Comment:
Hey, what is the meaning of `midstream` here ?

Reply:
@ludydoo These are downstream images. Should I change the name to `downstream`?

Reply:
Not necessarily, I was just wondering about what midstream means in this context?

Reply:
Honestly, we are using the term downstream/midstream interchangeably in this scenario. @robbycochran Do you have any preference for the naming scheme?